src/main/java/uk/gov/pay/api/app/config/RateLimiterConfig.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.pay.api.app.config;
+
+import io.dropwizard.Configuration;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+public class RateLimiterConfig extends Configuration {
+
+    @Valid
+    @NotNull
```

Comment:
`NotNull` with a primitive variable? And Min/Max any reason for limiting to these numbers ?